test/mempool-test.js

```diff
@@ -69,6 +75,28 @@ function dummyInput(script, hash) {
   return Coin.fromTX(fund, 0, -1);
 }
 
+async function getMockBlock(chain, txs = [], cb = true) {
+  if (cb) {
+    const raddr = KeyRing.generate().getAddress();
+    const mtx = new MTX();
+    mtx.addInput(new Input());
+    mtx.addOutput(raddr, 0);
+
+    txs = [mtx.toTX(), ...txs];
+  }
+
+  const now = Math.floor(Date.now() / 1000);
```

Comment:
There is a helper function for this

https://github.com/bcoin-org/bcoin/blob/d601b6a3031ea39190e1c1221b4a621b15b784e7/lib/utils/util.js#L58-L60

Reply:
This is a trivial one liner, it may make sense elsewhere, but I think is fine for the test case.